qemu/tests/timedrift.py

```diff
@@ -85,10 +87,13 @@ def restore_cpu_affinity(prev_masks):
     guest_load_command = params["guest_load_command"]
     guest_load_stop_command = params["guest_load_stop_command"]
     host_load_command = params["host_load_command"]
-    guest_load_instances = int(params.get("guest_load_instances", "1"))
-    host_load_instances = int(params.get("host_load_instances", "0"))
+    guest_load_instances = int(params["guest_load_instances"])
+    host_load_instances = int(params["host_load_instances"])
+    if not guest_load_instances or not host_load_instances:
```

Comment:
Will your definition in cfg file takes effect? When you have `host_load_instances = 0`, since here is `or`, even you define `guest_load_instances = 2`, it will be set to `vm.get_cpu_count()`, right?